grammar.js

```diff
@@ -247,7 +256,7 @@ module.exports = grammar({
         _external_body: ($) =>
             seq(
                 $.external_body,
-                NL,
+                repeat1(NL),
```

Comment:
kind of a drive-by fix: should be able to separate an external body from response handler scripts/response redirects with multiple newlines.

i.e. this should be valid, but isn't without `repeat1`:

```http
POST http://www.example.com
Content-Type: application/json

< ./body.json

> ./scripts/handler.json

>> response.json
```